types/migrations/20220428234639_add_morpheme_tag_description.sql

```diff
@@ -0,0 +1,36 @@
+-- Move the description column from abstract_morpheme_tag to morpheme_tag, so
+-- that morpheme tags in each system can have a distinct description.
+alter table morpheme_tag
+add column description text;
+
+update morpheme_tag
+set description = abstract_morpheme_tag.description
+from abstract_morpheme_tag
+where abstract_morpheme_tag.id = any(morpheme_tag.abstract_ids);
+
+alter table abstract_morpheme_tag
+drop column description;
+
+-- Add the Combine variant of SegmentType.
+alter type segment_type add value 'Combine';
```

Comment:
Hmm yeah I used 'Combine' because it means this morpheme will combine with the previous one, but it itself is not a combination. Perhaps a different name entirely would be better, any ideas?

Reply:
Since they are morphemes that will be added onto the end, best I think is"affix", since that would reflect its role as a morpheme going to be part of a larger word. Alternatively, if we want to keep things simple it might be sufficient to just end it in "-part" so its clear what its role is. For example, "CombinationPart" or "WordPart".

Reply:
I think there's some mixed terminology in your suggestion. In this context, morphemes are by definition word parts, so "WordPart" doesn't give us much semantically. Morphemes are always parts of larger words. "Affix" has incorrect implications too because many morphemes are considered affixes, having nothing to do with whether we internally combine them. "CombinationPart" could make sense, just feels like we're losing the forest through the trees with this approach to bikeshedding. We could call it a "Modifier", that could work, since it functions by adding extra nuance to the meaning of the previous morpheme.